llvm/test/CodeGen/AArch64/GlobalISel/phi-with-empty-type.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llc < %s -O0 -global-isel -mtriple=aarch64
```

Comment:
Should include checks (probably MIR checks, with -stop-after=irtranslator)

Reply:
Good idea. Included it.